### PR TITLE
Move RapidPro configuration to common/configuration

### PR DESCRIPTION
**Pull request by as2388**
This simplifies and improves the consistency in configuration, as well as making it easier to reuse rapid pro configuration in other tools in future

### CODE REVIEW

File: src/rapid_pro_to_engagement_db/configuration.py [hunk 1]
```diff
@@ -6,10 +6,3 @@ class FlowResultConfiguration:
     flow_name: str
     flow_result_field: str
     engagement_db_dataset: str
```
Comment (DanielMwendwa) [inline src/rapid_pro_to_engagement_db/configuration.py:8]: we should also move this to common/config

File: sync_rapid_pro_to_engagement_db.py [hunk 2]
```diff
@@ -4,7 +4,7 @@
 from core_data_modules.logging import Logger
 from engagement_database.data_models import HistoryEntryOrigin
 
-from src.common.configuration import UUIDTableConfiguration, EngagementDatabaseConfiguration
+from src.common.configuration import UUIDTableConfiguration, EngagementDatabaseConfiguration, RapidProConfiguration
 from src.rapid_pro_to_engagement_db.configuration import RapidProToEngagementDBConfiguration, FlowResultConfiguration
```
Comment (DanielMwendwa) [inline sync_rapid_pro_to_engagement_db.py:8]: rm RapidProToEngagementDBConfiguration

**Review by DanielMwendwa — APPROVED** [review]

File: sync_rapid_pro_to_engagement_db.py (hunk 2 above)
Comment (as2388) [reply sync_rapid_pro_to_engagement_db.py]: fixed, thanks

File: src/rapid_pro_to_engagement_db/configuration.py (hunk 1 above)
Comment (as2388) [reply src/rapid_pro_to_engagement_db/configuration.py:8]: I'm not sure we should - this configuration is specific to rapid pro -> db sync

Comment (DanielMwendwa) [reply src/rapid_pro_to_engagement_db/configuration.py:8]: yeah sure. you can go ahead and merge.

**Pull request merged**